platform/solr/solr-schema/src/main/resources/solr/conf/schema.xml

```diff
@@ -90,7 +90,7 @@
         </analyzer>
         <analyzer type="query">
             <tokenizer class="solr.WhitespaceTokenizerFactory"/>
-            <filter class="solr.SynonymGraphFilterFactory" synonyms="synonyms.txt" ignoreCase="true" expand="true"/>
+            <!-- <filter class="solr.SynonymGraphFilterFactory" synonyms="synonyms.txt" ignoreCase="true" expand="true"/> -->
```

Comment:
❓ should this simply be removed if not needed?

Reply:
Yes, it should probably just be removed.